src/components/navBar/navBar.css

```diff
@@ -0,0 +1,96 @@
+.navBar-container {
```

Comment:
Usually the convention is using kebab-case ( i.e navbar-container ) or camelCase ( i.e navBarContainer ) when naming classes, avoid using both together.
I would advise you to stick to kebab-case naming.